services/QuillLMS/Rakefile

```diff
@@ -2,7 +2,30 @@
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require File.expand_path('../config/application', __FILE__)
+require 'dotenv/tasks'
 
 EmpiricalGrammar::Application.load_tasks
 
 task default: [:spec]
+
+task after_hook: :dotenv do
+  post_slack_rake_task
+end
+
+Rake.application.tasks.reject do |task|
+  next if [Rake::Task['environment'], Rake::Task['dotenv'], Rake::Task['after_hook']].include?(task)
+
+  task.enhance([:after_hook])
+end
+
+def post_slack_rake_task
```

Comment:
It would be good to move this method out of the global namespace (it's a very specific method name, so it wouldn't collide, but just to keep the global namespace clean), e.g.
```ruby
module SlackTasks
  def post_slack_rake_task
  end
end
task after_hook: :dotenv do
  include SlackTasks
  post_slack_rake_task
end
```

Also, this might be premature, but you might consider moving this method out of tasks and making this reusable, e.g. 

```ruby
# create interface
Slack.post_to_developer(message)

# use (you can pass a ruby block to enhance)
task.enhance do
  Slack.post_to_developer(":rake: rake #{ARGV.join(' ')}")
end

Reply:
Great idea on moving out of global namespace. I will extract that out.  

I thought about extracting this out but the other scripts that post to slack function a little differently, calling rake locally `rake db:migrate` vs this PR where they are called remotely `heroku run rake db:migrate`.